x-pack/plugins/cases/public/components/user_action_tree/helpers.tsx

```diff
@@ -458,3 +461,55 @@ export interface Alert {
   signal: Signal;
   [key: string]: unknown;
 }
+
+export const getFirstItem = (items?: string | string[] | null): string | null => {
+  if (items == null) {
```

Comment:
nitpick: this kind of code can be confusing as `items` could be undefined and can also be null. I'd recommend the more verbose but clear and type safe way:

```
if (item === undefined || item === null) {
```